tests/integration-tests/tests/scaling/test_scaling.py

```diff
@@ -126,7 +126,7 @@ def test_scaling_with_manual_actions(scheduler, region, pcluster_config_reader,
 
     instance_ids = get_compute_nodes_instance_ids(cluster.cfn_name, region)
 
-    _test_replace_terminated_nodes(scheduler_commands, num_compute_nodes, instance_ids)
+    _test_replace_terminated_nodes(scheduler_commands, region, num_compute_nodes, instance_ids)
```

Comment:
the guidance i've gotten in the past is to just set the AWS_DEFAULT_REGION environ so that you don't have to pass the region arg around.